src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -117,3 +117,24 @@
   - name: Disable multiple features.
     text: az connectedk8s disable-features -n clusterName -g resourceGroupName --features custom-locations azure-rbac
 """
+
+helps['connectedk8s troubleshoot'] = """
+  type: command
+  short-summary: Collects diagnose infomation and  gets logs on the connected cluster.
+  parameters:
```

Comment:
what about --resource-group and --name

Reply:
Its there  in -h of troubleshoot.

Resource Id Arguments
    --ids               : One or more resource IDs (space-delimited). It should be a complete
                          resource ID containing all information of 'Resource Id' arguments. You
                          should provide either --ids or other 'Resource Id' arguments.
    **--name -n           : The name of the connected cluster.
    --resource-group -g : Name of resource group. You can configure the default group using `az
                          configure --defaults group=<name>`.**
    --subscription      : Name or ID of subscription. You can configure the default subscription
                          using `az account set -s NAME_OR_ID`.

Reply:
i get that. just wondering why we defining two parameters specifically here. In params.py, we already write the help for all parameters which then come up in the -h message. Wanted to know the use case here

Reply:
Got the point, will remove.